src/main.cpp

```diff
@@ -61,7 +61,23 @@ int main(int argc, char * argv[]) {
         args.printUsage();
         return 0;
     }
-
+    if(args.getSettingsPathSet()) {
+        QDir settingsPath(args.getSettingsPath());
+        qDebug() << "Settings Path Provided: " << settingsPath.absolutePath();
```

Comment:
you cannot qDebug() here, since it writes to mixxx.log which uses the settings Path()